src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
I would move **_Math.ceil(p.died / 100) === century_** to function;
Something like this ->
```
function calculateCentury(personDeath, century) {
 return Math.ceil(personDeath / 100) === century
}
```
I need some time to understand what here is happening, but new function will help me get it faster